### PR TITLE
Isuue #1754

### PR DESCRIPTION
**Pull request by AlexandrKhabarov**
# I have made things!

## Checklist

- [X] I have double checked that there are no unrelated changes in this pull request (old patches, accidental config files, etc)
- [X] I have created at least one test case for the changes I have made
- [X] I have updated the documentation for the changes I have made
- [X] I have added my changes to the `CHANGELOG.md`

### CODE REVIEW

**Comment by AlexandrKhabarov** [comment]
@sobolevn 
Hello!
Can you review my request, please?

File: tests/test_visitors/test_ast/test_classes/test_base_classes/test_kwargs_unpacking.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+from wemake_python_styleguide.violations.oop import UnpackingKwargsViolation
+from wemake_python_styleguide.visitors.ast.classes import WrongClassVisitor
+
+class_with_kwargs = """
+class Example(BaseTestClass, {0}):
+    '''Docs.'''
+"""
+
+
+@pytest.mark.parametrize('kwargs', ['a=1'])
```
Comment (sobolevn) [inline tests/test_visitors/test_ast/test_classes/test_base_classes/test_kwargs_unpacking.py:12]: Also:
- `some.ClassName`

**Review by sobolevn — APPROVED** [review]
Thanks a lot! Great work!

I will merge this PR into `0.16.0`

Can you please also try to fix win tests by skipping them (see #1891) in a **separate PR**?

**Comment by AlexandrKhabarov** [comment]
Closes #1754

**Comment by sobolevn** [comment]
Replaced by https://github.com/wemake-services/wemake-python-styleguide/pull/2217

**Comment by sobolevn** [comment]
Sorry, @AlexandrKhabarov, I've missed your PR! Thanks for working on it! 👍

**Pull request closed without merging**